spec/content.spec.js

```diff
@@ -18,12 +18,24 @@ describe('Content TestCase', function () {
     it('should removing paragraphs when a list is inserted inside of it', function () {
         this.el.innerHTML = '<p>lorem ipsum<ul><li>dolor</li></ul></p>';
         var editor = this.newMediumEditor('.editor', {
-            buttons: ['orderedlist']
-        }),
-            target = editor.elements[0].querySelector('p');
+                buttons: ['orderedlist']
+            }),
+            target = editor.elements[0].querySelector('p'),
+            range, sel;
         selectElementContentsAndFire(target);
         fireEvent(editor.toolbar.getToolbarElement().querySelector('[data-action="insertorderedlist"]'), 'click');
         expect(this.el.innerHTML).toMatch(/^<ol><li>lorem ipsum(<br>)?<\/li><\/ol><ul><li>dolor<\/li><\/ul>?/);
+
+        // for Chrome & Safari we manually moved the caret so let's check it
+        if (!isFirefox() && !isIE()) {
```

Comment:
Since `cleanListDOM` apply changes only for Chrome & Safari (as far as I understand from the commits history), the bug is only related to these browsers. That's why I restrict tests.